packages/tinybased/src/lib/TableBuilder.ts

```diff
@@ -99,10 +100,22 @@ export class TableBuilder<
     return this;
   }
 
+  defineKeyDelimiter(delimiter: string) {
+    this._keyDelimiter = delimiter;
+    return this;
+  }
+
   keyBy<TCellName extends OnlyStringKeys<TCells>>(
     cells: TCellName | TCellName[]
   ) {
     this._keys = Array.isArray(cells) ? cells : [cells];
     return this;
   }
+
+  composeKey(rowOrKeyValues: TCells | string[]) {
```

Comment:
Is there a way we could expose this somewhere so that we have an easy way to generate the key for a table in our application code based on the definition for the TableBuilder?

Reply:
In our application, I was thinking to use it from table, since each table would have its own custom delimiter, 
```
export const abcTable = new TableBuilder('abc_table')
  .add('id', 'string')
  .....
  
abcTable.composeKey([1, '2]);
 ```